modules/images/webp-uploads/helper.php

```diff
@@ -240,3 +240,35 @@ function webp_uploads_get_attachment_sources( $attachment_id, $size = 'thumbnail
 	// Return an empty array if no sources found.
 	return array();
 }
+
+/**
+ * Check whether the additional image is larger than the original image.
+ *
+ * @since n.e.x.t
+ *
+ * @param array $original An array with the metadata of the attachment.
```

Comment:
```suggestion
 * @param array $original   An array with the metadata of the attachment.
```

Reply:
align comment clocks

Reply:
Thank you for pointing it out. The lint command does not show any errors for this method. PR is updated with these changes.